src/sidebar/views/reports.ts

```diff
@@ -221,8 +221,30 @@ export class ReportsView implements TreeDataProvider<IssueMetadata> {
                             arguments: [this.currentFile, element.entryIndex]
                         }
                     },
-                    { ...element, description: '——' }
                 );
+
+                const checkerData = ExtensionApi.metadata.checkerData
+                    ?.get(`${entry.analyzer_name ?? ''}/${entry.checker_name ?? ''}`);
+
+                const docUrl = checkerData?.labels
+                    .find((val) => val.startsWith('doc_url:'))
+                    ?.substring(8);
+
+                if (docUrl !== undefined) {
```

Comment:
Is it possible that if no checker documentation is found, a "No documentation found" grayed-out button is displayed?